internals/overlord/state/identities.go

```diff
@@ -101,8 +109,25 @@ func (d *Identity) validateAccess() error {
 		}
 		gotType = true
 	}
+	if d.Cert != nil {
+		if d.Cert.Certificate == "" {
+			return errors.New("cert identity must specify certificate (PEM-encoded)")
+		}
+		block, rest := pem.Decode([]byte(d.Cert.Certificate))
+		if block == nil {
+			return errors.New("cert identity must have at least one valid PEM block")
```

Comment:
nit:

```identity user-1 invalid: certificate identity must include PEM encoded x509 certificate```

Can this not be the same error ? As it stands, it sounds like some PEM field exist in the cert identity, but what we mean is the certificate field is not encoded properly?